### PR TITLE
Add virsh cmd domdirtyrate-calc to virsh

### PR DESCRIPTION
**Pull request by iccaszhulili**
Signed-off-by: Lily Zhu <lizhu@redhat.com>

### CODE REVIEW

File: virttest/virsh.py [hunk 1]
```diff
@@ -4997,3 +4997,15 @@ def set_user_sshkeys(name, user, options="", **dargs):
     :return: CmdResult object.
     """
     return command("set-user-sshkeys %s %s %s" % (name, user, options), **dargs)
+
+
+def domdirtyrate_calc(name, options="", **dargs):
+    """
+    Calaulte vm's dirty page rate
+
+    :param name: VM name
```
Comment (chunfuwen) [inline virttest/virsh.py:5006]: miss options docstring

Comment (iccaszhulili) [reply virttest/virsh.py:5006]: @chunfuwen updated

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by kylazhang — APPROVED** [review]

File: virttest/virsh.py [hunk 2]
```diff
@@ -4997,3 +4997,16 @@ def set_user_sshkeys(name, user, options="", **dargs):
     :return: CmdResult object.
     """
     return command("set-user-sshkeys %s %s %s" % (name, user, options), **dargs)
+
+
+def domdirtyrate_calc(name, options="", **dargs):
+    """
+    Calaulte vm's dirty page rate
```
Comment (leidwang) [inline virttest/virsh.py:5004]: Hi @kylazhang ,The word "Calculate" is spelled incorrectly in this place, which will cause the CI check to fail. 
The error like this:
```
Run pylint --errors-only --disable=all --ignore=avocado-libs --enable=spelling --spelling-dict=en_US --spelling-private-dict-file=spell.ignore *
************* Module virttest.virsh
virttest/virsh.py:5008:0: C0402: Wrong spelling of a word 'Calaulte' in a docstring:
    Calaulte vm's dirty page rate
    ^^^^^^^^
Did you mean: ''Calculate''? (wrong-spelling-in-docstring)
Error: Process completed with exit code 16.
```

**Pull request merged**